test/e2e/wallet-helpers/vega-api.ts

```diff
@@ -86,13 +103,22 @@ export class VegaAPI {
   }
 
   private async executeListKeys() {
+    let keysArray: Key[] = []
     const keysString = await this.driver.executeScript<string>(async () => {
-      const keys = await window.vega.listKeys()
-      return JSON.stringify(keys)
+      try {
+        const keys = await window.vega.listKeys()
+        console.log('KEYS: ', keys)
```

Comment:
Do you want to keep these 'console.logs'?

Reply:
ah no, thanks